### PR TITLE
Code standards updated to PHP 7.1

### PR DESCRIPTION
**Pull request by TavoNiievez**
- The minimum version of PHP is now 7.1
- Added strict types
- Added return types
- Added some typehints
- Removed unnecessary qualifiers

### CODE REVIEW

File: src/Codeception/Lib/Connector/Laminas.php [hunk 1]
```diff
@@ -36,25 +56,22 @@ public function setApplicationConfig(array $applicationConfig)
...
      * @throws Exception
      */
-    public function doRequest($request)
+    public function doRequest($request): Response
     {
         $this->createApplication();
 
         $laminasRequest = $this->application->getRequest();
-        $uri            = new HttpUri($request->getUri());
+        $uri            = new Http($request->getUri());
```
Comment (Naktibalda) [inline src/Codeception/Lib/Connector/Laminas.php:66]: Http is a horrible and misleading class name here, could you leave the alias?

Comment (TavoNiievez) [reply src/Codeception/Lib/Connector/Laminas.php]: Done.

File: src/Codeception/Lib/Connector/Laminas.php [hunk 2]
```diff
@@ -101,40 +117,29 @@ public function doRequest($request)
...
         if (!$serviceManager->has($service)) {
-            throw new \PHPUnit\Framework\AssertionFailedError("Service $service is not available in container");
+            throw new AssertionFailedError(sprintf('Service %s is not available in container', $service));
         }
 
         return $serviceManager->get($service);
     }
 
-    public function persistService($name)
+    public function persistService($name): void
```
Comment (Naktibalda) [inline src/Codeception/Lib/Connector/Laminas.php:136]: $name is string

File: src/Codeception/Lib/Connector/Laminas.php [hunk 3]
```diff
@@ -101,40 +117,29 @@ public function doRequest($request)
...
     }
 
-    /**
-     * @param $name
-     * @param mixed  $service
-     *
-     * @return void
-     */
-    public function addServiceToContainer($name, $service)
+    public function addServiceToContainer($name, $service): void
```
Comment (Naktibalda) [inline src/Codeception/Lib/Connector/Laminas.php:142]: $name is string

**Review by Naktibalda — CHANGES_REQUESTED** [review]

**Comment by TavoNiievez** [comment]
@Naktibalda please merge this with `master` o secondary branch.
I don't have write access for this repo.

**Pull request merged**